packages/react/src/VideoPlayer/VideoPlayer.module.css

```diff
@@ -321,16 +315,16 @@
 
 .VideoPlayer__tooltip {
   position: absolute;
+  left: 50%;
   bottom: 100%;
   z-index: 2;
   width: 10px;
   height: 10px;
```

Comment:
Looks like there were quite a few hard coded px values here, could we get these cleaned up while we're at this refactor too please? Preference is to use rems where possible.

Reply:
Resolved in 4873e4713bad9c9ce1780008cf36c994d2f7053d